package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "commercetools-nodejs-boilerplate",
+  "version": "1.0.0-beta.1",
+  "description": "Generic Boilerplate for bootstrapping NodeJS applications",
+  "keywords": [
+    "commercetools",
+    "nodejs",
+    "boilerplate",
+    "starter"
+  ],
+  "homepage": "https://github.com/commercetools/nodejs-boilerplate#readme",
+  "bugs": {
+    "url": "https://github.com/commercetools/nodejs-boilerplate/issues"
+  },
+  "license": "MIT",
+  "contributors": [
+    "Abimbola Idowu <abimbola.idowu@commercetools.de> (https://github.com/hisabimbola)",
+    "Nicola Molinari <nicola.molinari@commercetools.de>",
+    "Philipp Sporrer <philipp.sporrer@commercetools.de>",
+    "Selwyn <talk@selwyn.cc> (https://selwyn.cc/)"
+  ],
+  "main": "./lib/main",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/commercetools/nodejs-boilerplate.git"
+  },
+  "scripts": {
+    "start": "npm run watch",
+    "test": "cross-env NODE_ENV=test babel-node test/*.spec.js test/*/**spec.js | tap-spec",
+    "test:watch": "watch \"npm run coverage:html\" src test -d",
+    "build": "npm run clean && babel src -d lib",
+    "clean": "rimraf lib/* && mkdirp lib",
+    "commit": "git-cz",
+    "codecov": "codecov",
+    "coverage": "npm run lint && cross-env NODE_ENV=test nyc --reporter=lcov --reporter=text npm test",
+    "coverage:html": "npm run lint && cross-env NODE_ENV=test nyc --reporter=html npm test",
+    "lint": "./scripts/lint src test",
+    "watch": "watch 'npm run test && npm run lint --silent' src bin"
+  },
+  "config": {
+    "commitizen": {
+      "path": "cz-conventional-changelog"
+    },
+    "ghooks": {
+      "commit-msg": "validate-commit-msg",
+      "pre-commit": "./scripts/lint-staged-files"
+    }
+  },
+  "eslintConfig": {
+    "extends": "commercetools"
+  },
+  "devDependencies": {
+    "babel-cli": "^6.14.0",
+    "babel-plugin-istanbul": "^2.0.1",
+    "babel-preset-es2015": "^6.14.0",
+    "codecov": "^1.0.1",
+    "commitizen": "^2.8.6",
+    "cross-env": "^2.0.1",
+    "cz-conventional-changelog": "^1.2.0",
+    "eslint-config-commercetools": "latest",
+    "ghooks": "^1.3.2",
+    "nyc": "^8.1.0",
+    "rimraf": "^2.5.4",
+    "tap-spec": "^4.1.1",
+    "tape": "^4.6.0",
+    "validate-commit-msg": "^2.8.0",
+    "watch": "^0.19.2"
+  },
+  "engines": {
+    "node": ">= 4.5.0"
+  },
+  "nyc": {
+    "include": [
+      "src/*.js"
+    ],
+    "sourceMap": false,
+    "instrument": false
+  }
+}
```

Comment:
We might consider having also a `package.json.template`, which contains placeholders for the library name and probably some common dependencies (babel, eslint, etc).

Or we keep the normal package.json but without much references to the boilerplate.